classes/controller/admin/answer/appdesk.ctrl.php

```diff
@@ -106,7 +106,7 @@ public function load_config()
 
                 // Gets header text
                 $headerText = \Arr::get($fieldAppdeskConfig, 'headerText', preg_replace('/\:\s*$/', ' ', $field->field_label));
-                $headerText = is_callable($headerText) ? $dataType($field) : $headerText;
+                $headerText = is_callable($dataType) ? $dataType($field) : $headerText;
```

Comment:
Are you sure of that ? I would have replaced `$dataType` with `$headerText` instead :
```
$headerText = is_callable($headerText) ? $headerText($field) : $headerText;
```

Reply:
You're right, it seems more logical.